src/components/box.js

```diff
@@ -36,6 +40,11 @@
     const opac = transparent ? 0 : 1;
     const [opacity, setOpacity] = useState(opac);
     const logic = useLogic(displayLogic);
+    const hasLink = linkType === 'internal' && linkTo && linkTo.id !== '';
+    const hasExternalLink =
+      linkType === 'external' && linkToExternal && linkToExternal.id !== '';
```

Comment:
And this to:
`const hasExternalLink = linkType === 'external' && linkToExternal?.id !== '';`